avocado/utils/software_manager.py

```diff
@@ -867,6 +867,20 @@ def provides(self, name):
         except process.CmdError:
             return None
 
+    def build_dep(self, name):
+        """Return True if build-dependencies are installed for provided package
```

Comment:
Hi @harish-24 , the code LGTM.

The only thing that I miss here is the PEP-0257 style. (missing an empty line after the first line).

Reply:
@beraldoleal Which is removed here https://github.com/avocado-framework/avocado/compare/c6157cc2c7eed9675f0c62f9126df091175e9552..4a6f0f35a1aa45d399943c0ac6218b55e7d97e18 (force-push link) due to inspekt style error in Travis.

Reply:
@harish-24 you removed the line. But the PEP-0257, says:

> Multi-line docstrings consist of a summary line just like a one-line docstring, **followed by a blank line**, followed by a more elaborate description.

Reply:
@beraldoleal If I dont remove the empty line I am getting this as pointed out by @clebergnu 
```
Tasks ended with 'fail': 003-selftests/inspekt-style.sh, 001-selftests/inspekt-indent.sh
```

So should I introduce the blank line which will lead to the above Travis CI failure? (but satisfies the PEP-0257).

Reply:
@harish-24 Unless I'm missing something, probably, the errors are not related.

An empty line after the first line should **not** trigger any linter issue. Can you please add this empty line and let travis complain so we can debug?

Reply:
@beraldoleal Not sure why, but seems travis has passed now with reverting the change, anyway Thanks.